CHANGES.rst

```diff
@@ -1,3 +1,702 @@
+  is an instance of the more general ``collections.abc.Mapping`` will work. [#12417]
+
+astropy.uncertainty
+^^^^^^^^^^^^^^^^^^^
+
+- Ensure that scalar ``QuantityDistribution`` unit conversion in ufuncs
+  works properly again. [#12471]
+
+astropy.units
+^^^^^^^^^^^^^
+
+- Add quantity support for ``scipy.special`` dimensionless functions
+  erfinv, erfcinv, gammaln and loggamma. [#10934]
+
+- ``VOUnit.to_string`` output is now compliant with IVOA VOUnits 1.0 standards. [#11565]
+
+- Units initialization with unicode has been expanded to include strings such as
+  'M☉' and 'e⁻'. [#11827]
+
+- Give a more informative ``NotImplementedError`` when trying to parse a unit
+  using an output-only format such as 'unicode' or 'latex'. [#11829]
+
+astropy.utils
+^^^^^^^^^^^^^
+
+- Fixed a bug in ``get_readable_fileobj`` that prevented the unified file read
+  interface from closing ASCII files. [#11809]
+
+- The function ``astropy.utils.decorators.deprecated_attribute()`` no longer
+  ignores its ``message``, ``alternative``, and ``pending`` arguments. [#12184]
+
+- Ensure that when taking the minimum or maximum of a ``Masked`` array,
+  any masked NaN values are ignored. [#12454]
+
+astropy.visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+- The tick labelling for radians has been fixed to remove a redundant ``.0`` in
+  the label for integer multiples of pi at 2pi and above. [#12221]
+
+- Fix a bug where non-``astropy.wcs.WCS`` WCS instances were not accepted in
+  ``WCSAxes.get_transform``. [#12286]
+
+- Fix compatibility with Matplotlib 3.5 when using the ``grid_type='contours'``
+  mode for drawing grid lines. [#12447]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- Enabled ``SlicedLowLevelWCS.pixel_to_world_values`` to handle slices including
+  non-``int`` integers, e.g. ``numpy.int64``. [#11980]
+
+
+Other Changes and Additions
+---------------------------
+
+- In docstrings, Sphinx cross-reference targets now use intersphinx, even if the
+  target is an internal link (``link`` is now ``'astropy:link``).
+  When built in Astropy these links are interpreted as internal links. When built
+  in affiliate packages, the link target is set by the key 'astropy' in the
+  intersphinx mapping. [#11690]
+
+- Made PyYaml >= 3.13 a strict runtime dependency. [#11903]
+
+- Minimum version of required Python is now 3.8. [#11934]
+
+- Minimum version of required Scipy is now 1.3. [#11934]
+
+- Minimum version of required Matplotlib is now 3.1. [#11934]
+
+- Minimum version of required Numpy is now 1.18. [#11935]
+
+- Fix deprecation warnings with Python 3.10 [#11962]
+
+- Speed up ``minversion()`` in cases where a module with a ``__version__``
+  attribute is passed. [#12174]
+
+- When ``astropy`` raises an ``OSError`` because a file it was told to write
+  already exists, then the error message now always suggests the use of the
+  ``overwrite=True`` argument. The wording is now consistent for all I/O formats. [#12179]
```

Comment:
```suggestion
- When ``astropy`` raises an ``OSError`` because a file it was told to write
  already exists, the error message now always suggests the use of the
  ``overwrite=True`` argument. The wording is now consistent for all I/O formats. [#12179]
```
shouldn't this be a "new feature" under `astropy.io` instead of a "other"?

Reply:
I'm not sure it's a new feature - it's just a change in error message? It's not even really an API change. I think 'other' is appropriate?

Reply:
See https://github.com/astropy/astropy/pull/12179#discussion_r729789830 for a third opinion.

Reply:
Thanks @eerovaher ! I tend to favor the API change section having now read that comment.

Reply:
Agree, will fix!